doc/tutorials/enable_ivshmem.rst

```diff
@@ -0,0 +1,151 @@
+.. _enable_ivshmem:
+
+Enable Inter-VM Communication Based on Ivshmem
+##############################################
+
+Users can use inter-vm communication based on ``ivshmem`` dm-land solution or
+hv-land solution according to the usage scenario and both solutions can be
+used at the same time. The ivshmem device using dm-land cannot communicate
+with the ivshmem device using hv-land. For more details about ivshmem inter-vm
+communication, please refer to :ref:`Ivshmem Inter-VM Communication High Level
+Design <ivshmem-hld>`
+
+Ivshmem Dm-land Usage
+*********************
+add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
+
+-  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the same
+   ``shm_name`` share a shared memory region. The ``shm_name`` needs to start
+   with ``sos:/`` prefix. For example, ``sos:/test``
```

Comment:
So hv-land uses a prefix name ``hv:/`` but dm-land uses ``sos:/``?  That's not user friendly

Reply:
The prefix is used to indicate the shared memory location.
For the hv-land, the shared memory is located at hypervisor
For the dm-land, the shared memory is located at service os

Reply:
I think @dbkinder means that the prefix `sos` isn't very user-friendly. I tend to agree with him. I would have preferred to see `svm` (for Service VM) or maybe `dm`?

Reply:
Yes. My comment indeed was about the inconsistency in naming, as Geoffroy noted. We've stopped using the term "SOS" and switched over to saying "Service VM" in all the documentation, so I was surprised to see ``sos:/`` here.

Reply:
OK, I will change it to dm and update source code for this.